modules/Users/User.php

```diff
@@ -678,12 +670,18 @@ public function save($check_notify = false)
         // set some default preferences when creating a new user
         $setNewUserPreferences = empty($this->id) || !empty($this->new_with_id);
 
+        if (!$this->verify_data()) {
+            SugarApplication::appendErrorMessage($this->error_string);
+            header('Location: index.php?action=Error&module=Users');
```

Comment:
Idk, is it possible here to use the proper function called SugarApplication::redirect() instead?